code/game/objects/items/singularityhammer.dm

```diff
@@ -97,16 +122,13 @@
 	target.throw_at(throw_target, 200, 4)
 	return
 
-/obj/item/twohanded/mjollnir/attack(mob/living/M, mob/user)
+/obj/item/mjollnir/attack(mob/living/M, mob/user)

```

Comment:
this would make more sense as a 2h attack sound var, the shocking itself i'm not sure about so I guess thats fine.

Reply:
not even in scope, you are now just nitpicking how other people did stuff on this PR.

Reply:
audio of 2hers is a part of this. you're refactoring 2handing behavior. It only makes sound when its 2handed. It's a big PR which is why people have avoided doing it.

Reply:
if two handed attacks have a different sound effect that should probably be a component arg

Reply:
added an attacksound when wielded, and converted this item